components/spell-check.R

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env Rscript
+#
+# Run spell check and save results
+
+library(magrittr)
+
+# Read in dictionary
+dictionary <- readLines(file.path('components', 'dictionary.txt'))
```

Comment:
Same thing here: I would make this an option, perhaps with this default location, but use `rprojroot` at a minimum.

Reply:
I was thinking I should do this but then I didn't. Good you mentioned it then.